src/footer/index.ts

```diff
@@ -0,0 +1,90 @@
+/*
+    This file was copied from mashlib/src/global/footer.ts file. It is modified to
+    work in solid-ui by adjusting where imported functions are found.
+ */
+import { IndexedFormula, NamedNode } from 'rdflib'
+import { authSession, currentUser } from '../authn/authn'
+import { addStyleClassToElement, getName, getPod, getPodOwner } from '../utils/headerFooterHelpers'
+
+const DEFAULT_SOLID_PROJECT_URL = 'https://solidproject.org'
```

Comment:
suggestion: I'm wondering since these were also in solid-panes, perhaps we could just export them from solid-ui and include them as well in solid-panes.

Reply:
They are the same for now but the logic allows you to customize them as you need.
Example: 
- someone used the header solid-ui elements for their app -> there are default settings BUT theoretically, in their own app they can define other links or menu names. 
- solid-panes is just a customizable version of the solid-ui  header elements/widgets